src/config/rt.rs

```diff
@@ -21,6 +22,19 @@ pub struct RtcBuild {
     pub final_dist: PathBuf,
     /// The directory used to stage build artifacts during an active build.
     pub staging_dist: PathBuf,
+    /// Optional pattern of the script to be injected instead of standard [default: None]
+    /// Should include {base}, {wasm}, {js}
+    pub pattern_script: Option<String>,
+    /// Optional pattern of the preload links to be injected instead of standard [default: None]
+    /// Should include {base}, {wasm}, {js}
+    pub pattern_preload: Option<String>,
+    /// Can be read only from config file
+    /// Optional parameters of replacements inside
+    /// While {var} is being replaced with the provided value,
+    /// {@path} is replaced with contents of the provided file.
+    /// This allows insertion of some big JSON state or even HTML files
+    /// as a part of the `index.html` build
```

Comment:
Let's update these docs to match the updates I've requested above.

Reply:
seems to be updated